crates/wasmtime/src/module.rs

```diff
@@ -82,6 +83,7 @@ struct ModuleInner {
     store: Store,
     compiled: CompiledModule,
     frame_info_registration: Mutex<Option<Option<Arc<GlobalFrameInfoRegistration>>>>,
+    stack_map_registration: Mutex<Option<Option<Arc<StackMapRegistration>>>>,
```

Comment:
Sort of continuing my comment from earlier, but this is an example where it would be great to not have this duplication. Ideally there'd be one "register stuff" call, although I'm not sure yet if this is fundamentally required to be two separate steps.